docs/examples/generate-jira-issues.yaml

```
@@ -0,0 +1,31 @@
+name: Generate Scorecards Reminders
+
+on:
+  schedule:
+    ## run every day at 9am
+    - cron: '0 9 * * *'
+  workflow_dispatch:
+
+jobs:
+    generate-scorecards-reminders:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Generate Jira Issues
+              uses: port-labs/port-sender@v0.1.14
```

I guess you will introduce a new version so we would want it to be your latest
Probably a new minor as this breaks current API
```suggestion
              uses: port-labs/port-sender@v0.2.0
```